Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -44,6 +59,28 @@ - (void)awakeFromNib {
     self.saveButton.tintColor = [UIColor wmf_blueTintColor];
     [self.saveButton setTitleColor:[UIColor wmf_blueTintColor] forState:UIControlStateNormal];
     self.saveButtonController.button = self.saveButton;
+
+    [self setupBlurViewAndLoadingIndicator];
+    [self unblurAndHideLoadingIndicator];
+}
+
+- (void)setupBlurViewAndLoadingIndicator {
+    UIBlurEffect* blurEffect = [[UIBlurEffect alloc] init];
+    self.blurView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
+    [self.contentView addSubview:self.blurView];
+    [self.contentView bringSubviewToFront:self.blurView];
+    [self.blurView mas_makeConstraints:^(MASConstraintMaker* make) {
+        make.edges.equalTo(self.contentView);
+    }];
+
+    self.activityIndicator = [[UIActivityIndicatorView alloc] init];
+    [self.activityIndicator setTranslatesAutoresizingMaskIntoConstraints:NO];
```

Comment:
this is unnecessary, masonry already sets this to `NO` when you call any of its make/remake/update methods.